.github/actions/macos_install/action.yml

```diff
@@ -7,4 +7,6 @@ runs:
       run: |
         brew install open-mpi
         brew install libomp
+        ln -s /usr/local/bin/gfortran-8 bin/gfortran
```

Comment:
Isn't the directory `/usr/local/bin/` in the search PATH? If that's the case I suggest you create the `gfortran` symbolic link in the same directory.

 On macOS `/usr/local/` is essentially reserved to Homebrew, which `/bin/` is a system directory.

Reply:
Yes, this was just a bad copy/paste